docs/domains/Domain.md

```diff
@@ -61,108 +59,100 @@ DomainStatus represents information about the status of a domain. Status may tra
+| `serverPod` | [Server Pod](#server-pod) | Customization affecting the generation of Pods for WebLogic Server instances. |
+| `serverService` | [Server Service](#server-service) | Customization affecting the generation of Kubernetes services for WebLogic Server instances. |
+| `serverStartPolicy` | string | The strategy for deciding whether to start a WebLogic server instance. Legal values are NEVER, or IF_NEEDED. More info: https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#starting-and-stopping-servers |
+| `serverStartState` | string | The WebLogic runtime state in which the server is to be started. Use ADMIN if server should start in the admin state. Defaults to RUNNING. |
 
 ### Configuration
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `introspectorJobActiveDeadlineSeconds` | number | The introspector job timeout value in seconds. If this field is specified it overrides the Operator's config map data.introspectorJobActiveDeadlineSeconds value. |
+| `introspectorJobActiveDeadlineSeconds` | number | The introspector job timeout value in seconds. If this field is specified then the Operator's ConfigMap data.introspectorJobActiveDeadlineSeconds value is ignored. |
 | `istio` | [Istio](#istio) | The Istio service mesh integration settings. |
 | `model` | [Model](#model) | Model in image model files and properties. |
-| `opss` | [Opss](#opss) | Configuration for OPSS security. |
-| `overrideDistributionStrategy` | string | Determines how updated configuration overrides are distributed to already running WebLogic servers following introspection when the domainHomeSourceType is PersistentVolume or Image.  Configuration overrides are generated during introspection from secrets, the overrideConfigMap field, and WebLogic domain topology. Legal values are DYNAMIC (the default) and ON_RESTART. See also introspectVersion. |
-| `overridesConfigMap` | string | The name of the config map for WebLogic configuration overrides. If this field is specified it overrides the value of spec.configOverrides. |
-| `secrets` | array of string | A list of names of the secrets for WebLogic configuration overrides or model. If this field is specified it overrides the value of spec.configOverrideSecrets. |
+| `opss` | [Opss](#opss) | Settings for OPSS security. |
+| `overrideDistributionStrategy` | string | Determines how updated configuration overrides are distributed to already running WebLogic servers following introspection when the domainHomeSourceType is PersistentVolume or Image.  Configuration overrides are generated during introspection from secrets, the overrideConfigMap field, and WebLogic domain topology. Legal values are DYNAMIC and ON_RESTART. See also introspectVersion. Defaults to DYNAMIC. |
+| `overridesConfigMap` | string | The name of the ConfigMap for WebLogic configuration overrides. If this field is specified then the value of spec.configOverrides is ignored. |
```

Comment:
If this field is specified then -> If this field is specified, then  (if...,(comma) then...)